### DISCUSSION TITLE
1.3.7 之后的 changelog 在哪？

### DISCUSSION THREAD

**Discussion by wygkzqa** [alibaba/formily#1413] [Q&A 1.x] [answered]
1.3.7 之后的 changelog 在哪？

**Comment by janryWang** [comment, marked as answer]
看commit记录，之前太忙，没来得及发realease文案

**Reply by janryWang** [reply]
后续2.0发布，会走自动化changelog

**Reply by wygkzqa** [reply]
升级到 1.3.15 后有一个问题，我有个组件是通过 InternalField 渲染的，并向该组件传递了 props，组件内部使用 state.props 后发现只有 index，并 没有 options，这是什么问题呢？

```
<ComponentA
  props={{
    index: 0,
    options: [...],
  }}
/>
```

**Reply by wygkzqa** [reply]
![image](https://user-images.githubusercontent.com/7590234/118087081-926f6500-b3f7-11eb-9758-178ca5e768b1.png)

options 莫名其妙就没了

**Reply by janryWang** [reply]
升级1.3.16
